### DISCUSSION TITLE
Load fonts from public folder

### DISCUSSION THREAD

**Discussion by eduardfossas** [webpack/webpack#13080] [Q&A]
Hello, hope you are having a nice day!

It is possible to declare a @font-face inside the source folder and this is loading a font from the public folder in Webpack 5. Like this:

@font-face {
font-family: 'Inter';
src: url('./assets/fonts/Inter.ttf');
}

Can someone explain me how?
Thanks in advance.

**Comment by alexander-akait** [comment]
I think the answer has already been, what's the problem?
